frontend/packages/modelServing/src/components/deployments/DeploymentsTableRow.tsx

```diff
@@ -24,7 +26,19 @@ export const DeploymentRow: React.FC<{
         {column.cellRenderer(deployment, column.field)}
       </Td>
     ))}
-    <Td dataLabel="Inference endpoint">-</Td>
+    <Td dataLabel="Inference endpoint">
+      <DeploymentEndpointsPopupButton
+        endpoints={deployment.endpoints}
+        loading={deployment.status?.state === InferenceServiceModelState.LOADING} // Not sure which states we need to show loading
```

Comment:
Not sure I understand this comment lol -- show loading state when it's loading?

Are you touching on this problem: https://issues.redhat.com/browse/RHOAIENG-25406?

If so, lets wait and do this for the main deployment as well as the rework -- don't make it unique to the refactor.